src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,46 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const handleTabSelect = (tab) => {
+    if (tab.id === selectedTabId) {
+      return;
+    }
+
+    onTabSelected(tab);
+  };
+
+  const getSelectedTab
```

Comment:
```suggestion
  const selectedTab
```

It's not a function, so "get" is redundant here